crates/core/src/contract/executor/runtime.rs

```
@@ -306,12 +306,18 @@ impl Executor<Runtime> {
             }
             ContractRequest::Update { key, data } => self.perform_contract_update(key, data).await,
             // FIXME
+            // Handle Get requests by returning the contract state and optionally the contract code
             ContractRequest::Get {
                 key,
                 return_contract_code,
             } => match self.perform_contract_get(return_contract_code, key).await {
-                Ok(_) => todo!(),
-                Err(_) => todo!(),
+                Ok((state, contract)) => Ok(ContractResponse::GetResponse {
+                    key,
+                    state: state.unwrap_or_else(|| WrappedState::new(Vec::new())),
```

mmmm I think we should return an error in this case?